src/driver/driver.js

```diff
@@ -51,6 +51,13 @@ export default class Driver {
     return driver.getDependencyTree(cwd, consumerPath, fullPaths, bindingPrefix);
   }
 
+
+  // TODO: Improve flow object return type
+  login(token: string, configPath: string, registryUrl: string): Object {
```

Comment:
maybe its better to call it npmLogin? or something like this? login is too general